src/components/pages/Friends/Friends.js

```diff
@@ -1,11 +1,61 @@
 import React from 'react';
+import smashRequests from '../../../helpers/data/smashRequests';
+import authRequests from '../../../helpers/data/authRequests';
+import FriendItem from '../../FriendItem/FriendItem';
+
 import './Friends.scss';
 
 class Friends extends React.Component {
+  state = {
+    users: [],
+    pendingFriends: [],
+    potentialFriends: [],
+    currentFriends: [],
+  };
+
+  componentDidMount() {
+    this.getAllUserFriends();
+  }
+
+  getAllUserFriends = () => {
+    smashRequests
+      .usersAndFriends(authRequests.getCurrentUid())
+      .then((users) => {
+        this.setState({ users });
```

Comment:
you can remove this setState if we don't need to set users state

Reply:
Fixed in https://github.com/MCrank/react-nutshell/pull/13/commits/3c56bc4e76dc5e8953726da354218a30ac961889